docs/excel/excel-data-types-overview.md

```diff
@@ -16,7 +16,12 @@ Prior to the data types addition, the Excel JavaScript API supported string, num
 
 Data types expand Excel JavaScript API support beyond the four original data types (string, number, boolean, and error) to include [web images](excel-data-types-concepts.md#web-image-values), [formatted number values](excel-data-types-concepts.md#formatted-number-values), [entities](excel-data-types-concepts.md#entity-values), arrays within entities, and improved [error data types](excel-data-types-concepts.md#improved-error-support) as flexible data structures. These types, which power many [linked data types](https://support.microsoft.com/office/what-linked-data-types-are-available-in-excel-6510ab58-52f6-4368-ba0f-6a76c0190772) experiences, allow for precision and simplicity during add-in calculations and extend the potential of Excel add-ins beyond a 2-dimensional grid.
 
-To learn how to use data types APIs, start with the [Excel data types core concepts](excel-data-types-concepts.md) article.
+Take the following steps to complete the data types learning path.
+
+1. Learn data types API basics in the [Excel data types core concepts](excel-data-types-concepts.md) article.
+1. Install [Script Lab](../overview/explore-with-script-lab.md) in Excel and explore the **Data types** section in our **Samples** library.
+1. Learn how entity data types extend Excel add-ins beyond a 2-dimensional grid in the [Use cards with entity value data types](excel-data-types-entity-card.md) article.
+1. Try out the [Create and explore data types in Excel](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-data-types-explorer) sample in our [OfficeDev/Office-Add-in-samples](https://github.com/OfficeDev/Office-Add-in-samples) repository.
 
 > [!NOTE]
 > To start experimenting with data types right away, install [Script Lab](../overview/explore-with-script-lab.md) in Excel and check out the **Data types** section in our **Samples** library. You can also explore the Script Lab samples in our [OfficeDev/office-js-snippets](https://github.com/OfficeDev/office-js-snippets/tree/prod/samples/excel/20-data-types) repository.
```

Comment:
The See also section now feels redundant. Is there value in removing it, and instead, giving you previous list a header like "Resources"?